NGitLab/IPipelineClient.cs

```diff
@@ -102,5 +102,18 @@ public interface IPipelineClient
         /// </summary>
         /// <param name="pipelineId">ID of the pipeline</param>
         TestReport GetTestReports(int pipelineId);
+
+        /// <summary>
+        /// Returns the bridges of a pipeline.
+        /// </summary>
+        /// <param name="query"></param>
+        IEnumerable<Bridge> GetBridges(PipelineJobQuery query);
```

Comment:
This method can be removed. `GetBridgesAsync` can be enumerated synchronously. Indeed, `GitLabCollectionResponse` implements both `IEnumerable` and `IAsyncEnumerable`.